ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6623,69 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+SEDPM:
+  description: "Supply, exhaust pressure monitoring"
+  is_abstract: true
+  uses:
+  - supply_air_differential_pressure_sensor
+  - exhaust_air_differential_pressure_sensor
+  implements:
+  - MONITORING
+
+DXC:
```

Comment:
Each compressor should have its own run status. Flag as missing for your unit if you don't see them rather than creating a new type.